OpenCL_C.txt

```diff
@@ -22,10 +22,15 @@ Khronos{R} OpenCL Working Group
 
 :cpp: pass:[C++]
 :cpp11: pass:[C++11]
+:kernel: pass[__kernel]
+:read_only: pass[__read_only]
+:write_only: pass[__write_only]
+:read_write: pass[__read_write]
```

Comment:
```suggestion
:kernel: pass:[__kernel]
:read_only: pass:[__read_only]
:write_only: pass:[__write_only]
:read_write: pass:[__read_write]
```
These missing `:` are the reason that the raw pass text is in the rendered specification.